ciphey/basemods/Decoders/base58_flicker.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
+import base58
+
+
+@registry.register
+class Base58_flickr(Decoder[str, str]):
```

Comment:
Change the filename so it's the same as the class name to avoid confusion

Reply:
The reason I did that was I thought that all the filenames are supposed to be in lowercase, no?

Reply:
Never mind, I see I made a typo in the filename. I fixed it.